antora-ui-camel/src/css/vars.css

```diff
@@ -10,6 +10,7 @@
   --color-gray-30: #8e8e8e;
   --color-gray-50: #808080;
   --color-gray-70: #5d5d5d;
+  --color-gray-90: #555;
```

Comment:
This is 65% grey not 90%, perhaps we can use 70% instead of introducing this

Reply:
**color-gray-70** is being used. I could make use of **color-gray-60** for this one, I feel.